src/SDKs/Batch/DataPlane/Tools/ObjectModelCodeGeneration/ObjectModelCodeGenerator/ModifiableClassTemplate.cs

```diff
@@ -93,9 +93,9 @@ public virtual string TransformText()
             this.Write(this.ToStringHelper.ToStringWithCulture(kvp.Key.PropertyContainerPropertyName));
             this.Write(" = this.CreatePropertyAccessor(\r\n                    ");
             this.Write(this.ToStringHelper.ToStringWithCulture(protocolObjectGetter));
-            this.Write(",\r\n                    \"");
+            this.Write(",\r\n                    nameof(");
             this.Write(this.ToStringHelper.ToStringWithCulture(kvp.Key.Name));
-            this.Write("\",\r\n                    ");
+            this.Write("),\r\n                    ");
```

Comment:
@matthchr would you mind double-checking that this is the string you wish to output? It appears to be different than before.

Reply:
Yeah, this is right -- I changed it to use nameof rather than hardcode the name, so we don't have escaped quote (`\"`) characters anymore.